src/main/java/meteordevelopment/meteorclient/mixin/ChatHudMixin.java

```diff
@@ -163,22 +167,20 @@ private static double getMessageOpacityMultiplier(int age) {
 
     private void drawIcon(MatrixStack matrices, String line, int y, float opacity) {
         if (METEOR_PREFIX_REGEX.matcher(line).find()) {
-            RenderSystem.setShaderTexture(0, METEOR_CHAT_ICON);
             matrices.push();
             RenderSystem.setShaderColor(1, 1, 1, opacity);
```

Comment:
This probably is no longer needed, since drawContext is meant to take over a lot of the RenderSystem calls